docs/source/opens-upgrade.rst

```diff
@@ -87,27 +87,16 @@ instead.)
 Ubuntu 14.04
 ^^^^^^^^^^^^
 
-First, upgrade packaged components::
+First, use ``apt-get`` to install the updated packages.  On each compute host
+you can upgrde only the Calico packages, as follows::
 
     apt-get update
-    apt-get dist-upgrade
+    apt-get install calico-compute calico-felix calico-common python-etcd
```

Comment:
The other packages should get pulled in by the version dependencies in the "main" packages.  Any package that you explicitly `apt-get install` gets set to "manually installed" in the APT database, which we don't want for all our dependencies.

I've included felix and common here because I wasn't sure that calico-compute had an explicit version dependency in it.  I included python-etcd because we've made 0.4.3 available but our dependency is still 0.4.1 so it wouldn't get upgraded by default.